test/unit/spec/v2/view-builder/views/AuthenticatorIdpVerifyView_spec.js

```diff
@@ -0,0 +1,132 @@
+import AuthenticatorIdPEnrollView from 'v2/view-builder/views/idp/AuthenticatorIdPEnrollView';
+import AppState from 'v2/models/AppState';
+import Settings from 'models/Settings';
+import $sandbox from 'sandbox';
+import IdpAuthenticatorVerifyResponse from '../../../../../../playground/mocks/data/idp/idx/authenticator-verification-idp.json';
+import IdpAuthenticatorEnrollResponse from '../../../../../../playground/mocks/data/idp/idx/authenticator-enroll-idp.json';
+
+describe('v2/view-builder/views/idp/AuthenticatorIdPVerifyView', function() {
+  let testContext;
+  let appStateTriggerSpy;
+  beforeEach(function() {
+    testContext = {};
+    testContext.init = (
+      requestName = 'Verification',
+      skipIdpFactorVerification = false,
+      isFailure = false
+    ) => {
+      let isVerify = requestName === 'Verification';
+      let currentAuthenticator =  isVerify ? IdpAuthenticatorVerifyResponse.currentAuthenticator.value :
+        IdpAuthenticatorEnrollResponse.currentAuthenticator.value;
+      let authenticatorEnrollments = [];
+      let app = {};
+      const appState = new AppState({
+        currentAuthenticator,
+        authenticatorEnrollments,
+        app
+      }, {});
+
+      if (isFailure) {
+        appState.set('messages', {
+          'value': [
+            {
+              'message': 'Authentication failed. Please try again.',
+              'links': [
+                {'label': 'Help link 1', 'url': 'https://www.okta.com/'},
+                {'label': 'Help link 2', 'url': 'https://www.okta.com/help?page=1'}
+              ],
+              'i18n': {
+                'key': 'security.access_denied_custom_message'
+              },
+              'class': 'ERROR'
+            }
+          ]
+        });
+      }
+
+      jest.spyOn(appState, 'getRemediationAuthenticationOptions').mockReturnValue(formName => {
+        if (formName === 'select-authenticator-authenticate') {
+          return [ { label: 'some authenticator '} ];
+        }
+        return [];
+      });
+
+      const settings = new Settings({ baseUrl: 'http://localhost:3000', 'features.skipIdpFactorVerificationBtn': skipIdpFactorVerification });
+      const currentViewState = {
+        name: 'challenge-authenticator',
+        relatesTo: {
+          value: currentAuthenticator,
+        },
+      };
+      const ViewClass = isVerify ? AuthenticatorIdPVerifyView : AuthenticatorIdPEnrollView;
+      testContext.view = new ViewClass({
+        el: $sandbox,
+        appState,
+        settings,
+        currentViewState,
+      });
+      appStateTriggerSpy = jest.spyOn(testContext.view.options.appState, 'trigger');
+      testContext.view.render();
+    };
+  });
+
+  afterEach(function() {
+    $sandbox.empty();
+  });
+
+  it.each(['Verification', 'Enrollment']) ('shows idp authenticator %s screen with ' +
+        'SKIP_IDP_FACTOR_VERIFICATION_BUTTON disabled', function(requestName) {
```

Comment:
in this repo there is no concept of `SKIP_IDP_FACTOR_VERIFICATION_BUTTON`, so instead let's change the description of all the tests to refer to `features.skipIdpFactorVerificationBtn`